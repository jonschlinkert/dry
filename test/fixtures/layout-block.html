{% layout "layout-default.html" %}

{% block "footer" %}
footer: "fixtures/layout.html"
{% endblock %}
