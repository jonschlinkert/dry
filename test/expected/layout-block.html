<!DOCTYPE html>
<html lang="en">
  
  <head>
    <meta charset="UTF-8">
    <title>Body fixture</title>
  </head>
  
  <body>
    


    
footer: "fixtures/layout.html"
  </body>
</html>